src/components/inline-link/index.tsx

```diff
@@ -22,24 +20,24 @@ const InlineLink = ({
 	)
 
 	/**
-	 * This case is possible for some of the extra <a> elements that are added by
-	 * our `anchor-links` remark plugin.
+	 * An undefined "href" is possible for some of the extra "target" <a> elements
+	 * that are added by our `anchor-links` remark plugin. These elements have
+	 * an "id", and act as a target for anchor links, but do not have an href.
+	 *
+	 * For now, we still want to render these elements, as they're needed
+	 * for some anchor links to function.
 	 *
 	 * FUTURE TODOs
 	 *   - update the plugin to no longer add empty <a> elements
-	 *   - update this to throw an error
+	 *   - update this to throw an error if href is falsy
+	 *
+	 * Asana task:
+	 * https://app.asana.com/0/1100423001970639/1202488740132127/f
```

Comment:
This is the link to the new task, can we reference it instead?

```suggestion
	 * https://app.asana.com/0/1100423001970639/1202560954440296/f
```